pkg/ddl/create_table.go

```diff
@@ -274,7 +274,7 @@ func (w *worker) onCreateTables(jobCtx *jobContext, job *model.Job) (int64, erro
 		return ver, errors.Trace(err)
 	}
 	createTablesEvent := notifier.NewCreateTablesEvent(tableInfos)
```

Comment:
I prefer we constuct many CreateTableEvent (not "s"), and call `asyncNotifyEvent` one-by-one.

The good point of current code is, it can unify the logic of multi-schema change and merged DDL. But the bad point is we have a temporary `ActionCreateTables` type event but never deliver it. Developers and subscribers may be confused about this special case when checking all possible types.

Reply:
update in [4ea3c0e](https://github.com/pingcap/tidb/pull/56739/commits/4ea3c0e644b0fd275c14bf627e6dc75d059ed7fd), ptal